src/components/signupform/SignupForm.js

```diff
@@ -116,7 +111,7 @@ function Form() {
           )}
         </div>
         <div className="mb-2">
-          <input type="checkbox" name="acceptTerms" id="acceptTerms" />
+          <input type="checkbox" name="acceptTerms" id="acceptTerms" required />
```

Comment:
@TaylorDJones11 When the user selects this box, does the fact that they chose it actually get saved in the user record? 👀  Or is it window dressing right now? (which is fine, zero judgement)